src/Commands/ImportDump.php

```diff
@@ -92,18 +98,40 @@ public function handle()
             return;
         }
 
-        if (!($optionRemote || $optionFile || $optionDump)) {
+        if (!($optionRemote || $optionFile || $optionDump || $optionLatest)) {
             if ($this->choice('Do you want to download and import a fresh dump from the server or an existing local dump?',
                     [
-                        '1' => 'Download remote dump',
-                        '2' => 'Import existing local dump',
+                        1 => static::DOWNLOAD_REMOTE_DUMP,
+                        2 => static::IMPORT_EXISTING_LOCAL_DUMP,
                     ],
-                    'Download remote dump') == 'Download remote dump') {
+                    1) == static::DOWNLOAD_REMOTE_DUMP) {
                 $optionRemote = true;
-            };
+            }
+        }
+
+        if ($optionLatest) {
+            try {
+                $importFilePath = $protector->getLatestDumpName();
+            } catch (FileNotFoundException $fileNotFoundException) {
+                if (!$optionRemote) {
+                    $this->error($fileNotFoundException->getMessage());
+                    return;
+                } else {
+                    $this->warn(sprintf('There are no files in %s', $disk->path($basePath)));
+                }
+            }
+        } elseif ($optionFile || $optionDump) {
+            if ($disk->exists($relativePath)) {
+                $importFilePath = $relativePath;
+            } elseif (!$optionRemote) {
+                $this->error((new FileNotFoundException($relativePath))->getMessage());
+                return;
+            } else {
+                $this->warn(sprintf('File not found: %s', $relativePath));
+            }
         }
 
-        if ($optionRemote) {
+        if (!$importFilePath && $optionRemote) {
```

Comment:
How does the remote import start if the user selected (1) in line 102+? `$optionRemote` will be `true`, but `$importFilePath` will be `null`, if I didn't miss anything?

This would mean even if the user selected that they want to import a remote dump, it will ask them to choose a local dump...?

Reply:
> `$optionRemote` will be `true`, but `$importFilePath` will be `null`, if I didn't miss anything?

That's the whole point, isn't it? The remote option is supposed to be a fallback for the other options when used together, so when those didn't work despite being stated, `$importFilePath` will still be `null` and it will download a remote dump instead.